app/models/user_access.rb

```diff
@@ -66,15 +66,17 @@ def accessible_facilities(action)
   end
 
   def accessible_admins(action)
+    return User.none unless action == :manage
```

Comment:
I feel like this statement is enough to "gate" this function. Do we still need,

```ruby
return User.none unless action_to_level(:manage).include?(user.access_level.to_sym)
```

...down below?

Reply:
Yup, this is still necessary. This actually checks whether the `current_admin`'s `access_level` allows them to perform `:manage` actions, the first guard clause only checks that the action is `:manage`.

I've added that check separately so that power_user don't get back data for non `:manage` actions on these methods